### DISCUSSION TITLE
Provide a preset to enable updating the go version

### DISCUSSION THREAD

**Discussion by eriksw** [renovatebot/renovate#23922] [Suggest an Idea]
### Tell us more.

I have a bunch of repos that are tools that I want to be kept up to date with the most recent version of go. Renovate supports this, but it's off by default as of https://github.com/renovatebot/renovate/pull/19453

It would be nice if the [package rule](https://github.com/renovatebot/renovate/blob/719354793f970de9860a8b78ef34c793657fe3b4/lib/modules/versioning/go-mod-directive/readme.md) to enable updating the version of go was a built-in preset.

```json
{
  "packageRules": [
    {
      "matchDatasources": ["golang-version"],
      "rangeStrategy": "bump"
    }
  ]
}
```

Additionally, having it be in the documentation (in the list of built-in presets) would improve discoverability of the fact that Renovate is capable of keeping the version of go up to date. (As an anecdote: A _lot_ of Renovate's most useful features I only found out about by reading through the list of available presets in the documentation.)

**Comment by rarkins** [comment]
PR welcome - it can be called `:bumpGoDirective`
